diff.txt

```diff
@@ -0,0 +1,51 @@
+diff --git a/spring-integration-ip/src/main/java/org/springframework/integration/ip/tcp/connection/TcpNioConnection.java b/spring-integration-ip/src/main/java/org/springframework/integration/ip/tcp/connection/TcpNioConnection.java
```

Comment:
This file must not be here.

Reply:
how can i remove it?

Reply:
It must be somehow in your local project copy.
So, just remove it  and push commit.